kano_settings/system/display.py

```diff
@@ -263,6 +263,13 @@ def is_mode_fallback():
     return w == 640 and h == 480
 
 
+def is_screen_kit():
+    '''
+    Returns True if the screen is a Kano Screen Kit.
+    '''
+    return get_model() in ('ADA-HDMI', 'MST-HDMI1')
```

Comment:
Looking at how `get_model` is implemented, I think it's quite odd. Could you please use `get_edid_name(use_cached=False)` ?